archive/n/nim/fibonacci.nim

```diff
@@ -0,0 +1,29 @@
+# Fibonacci Sample Program in Nim
+# Input the number of iterations as a command-line parameter
+import os
+import strutils
+
+var n: BiggestUInt
+try:
+    n = paramStr(1).parseBiggestUInt
+except IndexError:
+    echo "Usage: please input the count of fibonacci numbers to output"
+    quit(1)
+except ValueError:
+    echo "invalid input: not a number"
```

Comment:
We use the same string for both cases. Sorry if that was confusing in the table.

```suggestion
    echo "Usage: please input the count of fibonacci numbers to output"
```

Reply:
Updated